### DISCUSSION TITLE
How to prevent model from using RAM when offloading full to GPU?

### DISCUSSION THREAD

**Discussion by Simplegram** [abetlen/llama-cpp-python#1490] [Q&A] [answered]
How do I load a GGUF only on VRAM and not on RAM? In the original llama.cpp, a model only loads to VRAM, completely leaving RAM empty. I can load models as large as the combined size of my RAM and VRAM. I want to use llama-cpp-python because llama.cpp's API endpoint is not compatible with llamaindex's ReAct agent. Thanks in advance!

**Comment by yassinz** [comment, marked as answer]
Change parameter use_mmap=False

**Reply by Simplegram** [reply]
Oh, my dumb mistake! I thought the parameter didn't work lol. I used it this way ```--use_mmap False``` before and it didn't change anything. Thank you.
